libstorage/CachedStorage.cpp

```diff
@@ -99,10 +99,12 @@ void Cache::setTableInfo(TableInfo::Ptr tableInfo)
     m_tableInfo = tableInfo;
 }
 
-CachedStorage::CachedStorage()
+CachedStorage::CachedStorage(dev::GROUP_ID const& _groupID)
 {
+    m_groupID = _groupID;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'm_groupID' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/app/fisco/FISCO-BCOS/pullRequest?prid=4619929)